src/VisualStudio/Core/Def/Implementation/AnalyzerDependency/AnalyzerDependencyCheckingService.cs

```diff
@@ -202,6 +203,27 @@ private Task<AnalyzerDependencyResults> GetConflictsAsync()
             return _task;
         }
 
+        private static IEnumerable<AssemblyIdentity> GetExplicitlyIgnoredAssemblyIdentities()
+        {
+            // Microsoft.VisualBasic.dll
+            var list = new List<AssemblyIdentity>();
+            AddAssemblyIdentity(list, "Microsoft.VisualBasic, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a");
```

Comment:
Because of downtargeting I would think we shouldn't include versions. Also, I'm not sure how the version numbers are for portable versions/platforms/mono/etc.